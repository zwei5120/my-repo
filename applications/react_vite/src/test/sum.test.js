import {
  describe,
  it
} from 'vitest'

describe('test this sum file', () => {
  it('test sum', ({ expect }) => {
    expect(1 + 2).toMatchSnapshot()
  })
})