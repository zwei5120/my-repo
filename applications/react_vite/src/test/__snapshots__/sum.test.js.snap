// Vitest Snapshot v1

exports[`test this sum file > test sum 1`] = `2`;
